### ISSUE TITLE
Test `legendIsQuestionTitle` for accessibility

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2637]
The DS is lacking guidance on when you should/should not use `legendIsQuestionTitle`. In the [onsQuestion](https://service-manual.ons.gov.uk/design-system/patterns/question) macro, you can't use it when the question requires description, definition and guidance before the fieldset. 

This was designed based on the assumption the inputs should be announced immediately after the legend by screen readers, to avoid confusion. This assumption needs testing, and guidance added to the docs when it's clear when it should be used.

**Comment by yatesn** [comment]
Raise on Content Design board

**Comment by daniellecorke** [comment]
I will ask Vicki for an update on this one.

**Comment by daniellecorke** [comment]
@JNATION85 this ticket is not on the centralised UCD board - is it still needed? If not, we can close it. If so, I'll add it to the UCD board.

**Comment by daniellecorke** [comment]
Discussed with Joe - I will add this to the UCD Board.

**Comment by daniellecorke** [comment]
On the UCD Board: https://jira.ons.gov.uk/browse/CU-90.
